<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>668</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>371</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="3">
      <widget class="QPushButton" name="pushButton_0">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_acos">
       <property name="text">
        <string>acos()</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_plus">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="pushButton_equal">
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_asin">
       <property name="text">
        <string>asin()</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QPushButton" name="pushButton_undo">
       <property name="text">
        <string>UNDO</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_ln">
       <property name="text">
        <string>ln()</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="6">
      <widget class="QLineEdit" name="expressionWindow">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border-bottom: 1px solid gray;
}

background-color: white;</string>
       </property>
       <property name="text">
        <string>  0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="pushButton_9">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_log">
       <property name="text">
        <string>log()</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_atan">
       <property name="text">
        <string>atan()</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="pushButton_power">
       <property name="text">
        <string>^</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_sin">
       <property name="text">
        <string>sin()</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_div">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_lbracket">
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_1">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QLineEdit" name="xValue">
       <property name="text">
        <string>  0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="pushButton_dot">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_tan">
       <property name="text">
        <string>tan()</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_mult">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3" colspan="3">
      <widget class="QPushButton" name="pushButton_PLOT">
       <property name="text">
        <string>PLOT</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButton_mod">
       <property name="text">
        <string>mod</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_minus">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_sqrt">
       <property name="text">
        <string>sqrt()</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_rbracket">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pushButton_x">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_cos">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>cos()</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>max x:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="maxX">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>min x:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="minX">
       <property name="text">
        <string>-10</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>max y:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="maxY">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="minY">
       <property name="text">
        <string>-10</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>min y:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>588</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMODE">
    <property name="title">
     <string>MODE</string>
    </property>
    <addaction name="actionCreditCalculator"/>
    <addaction name="actionDepositCalculator"/>
   </widget>
   <addaction name="menuMODE"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreditCalculator">
   <property name="text">
    <string>CreditCalculator</string>
   </property>
  </action>
  <action name="actionDepositCalculator">
   <property name="text">
    <string>DepositCalculator</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
